src/guide-start.md

```diff
@@ -63,7 +63,29 @@ All generators implement the [`Rng`] trait, which provides the [`gen`],
 Rand provides functionality on iterators and slices via two more traits,
 [`IteratorRandom`] and [`SliceRandom`].
 
+## Fixed seed RNGs
 
+You may have noticed the use of `thread_rng()` above and wondered how to
+specify a fixed seed. To do so, you need to specify an RNG then use a method
+like [`seed_from_u64`] or [`from_seed`].
+
+We use `ChaCha8Rng` below because it is fast and portable with good quality.
+See the [RNGs] section for more RNGs, but avoid `SmallRng` and `StdRng` if you
+care about reproducible results.
+
+```rust,editable
+extern crate rand;
+extern crate rand_chacha;
+
+use rand::{Rng, SeedableRng};
+
+fn main() {
+    let mut rng = rand_chacha::ChaCha8Rng::seed_from_u64(10);
```

Comment:
Maybe mention that this is not good enough if the RNG is supposed to be unpredictable by observing some samples.

Reply:
It is good enough, as long as you don't need cryptographic strength. This section *really* isn't aimed at anyone writing cryptographic code; perhaps that should be clearer (though it should be the moment one looks at the `seed_from_u64` docs, which are linked).

Reply:
Yes, but the conservative approach would be to suggest the secure method by default. I think it should at least be clarified.

> though it should be the moment one looks at the seed_from_u64 docs, which are linked

It is likely that someone will just copy and paste without consulting the documentation.

Reply:
Disclaimer added. I don't wish to adjust this example for security since (a) it requires retrieving a seed from somewhere and (b) is less appropriate for I would guess nearly everyone copying the example: use of a pre-seeded CSPRNG is not something to be learned from a quick-start guide.